nrkdl.py

```diff
@@ -473,7 +475,15 @@ def _console(self, q):
         else:
             # use reverse since 0 is the closest match and i dont want to scoll
             for i, hit in reversed(list(enumerate(response['hits']))):
-                print('{0:>3}: {1}'.format(i, c_out(hit['hit']['title'])))
+                if 'category' in hit['hit']:
+                    category = c_out(hit['hit']['category']['title'])
+                else:
+                    category = ""
+                print('{0:>3}: {1} [{2}]'.format(i, c_out(hit['hit']['title']), category))
```

Comment:
Can you make sure that [] isnt printed if the categoey is missing. Just make it part of the categoey str or something